src/server/service/kv.rs

```diff
@@ -2296,6 +2296,7 @@ fn needs_reject_raft_append(reject_messages_on_memory_ratio: f64) -> bool {
         if (raft_msg_usage + cached_entries + applying_entries) as f64
             > usage as f64 * reject_messages_on_memory_ratio
         {
+            // FIXME: this doesn't output to logfile.
```

Comment:
Something wrong with `slog_global`?

Reply:
I don't know. I only know this branch is taken but no log is found.